hack/testing-olm/utils

```diff
@@ -13,74 +13,6 @@ gather_logging_resources() {
   local runtime=${3:-$(date +%s)}
   outdir=$outdir/$runtime
   mkdir $outdir ||:
-  oc -n ${LOGGING_NS} get configmaps -o yaml > $outdir/configmaps.yaml 2>&1 || :
-  oc -n ${LOGGING_NS} get secrets -o yaml > $outdir/secrets.yaml 2>&1 || :
-  oc -n ${LOGGING_NS} get cronjobs -o yaml > $outdir/cronjobs.yaml 2>&1 || :
-  
-  oc -n ${LOGGING_NS} get deployments -o wide > $outdir/deployments.txt 2>&1 ||    :
-  oc -n ${LOGGING_NS} get ds -o wide > $outdir/daemonsets.txt 2>&1 ||    :
-  oc -n ${LOGGING_NS} get pods -o wide > $outdir/pods.txt 2>&1 || :
-  oc -n ${LOGGING_NS} get clusterlogging instance -o yaml > $outdir/clusterlogging.instance.yaml  ||:
-  # Note: clusterlogforwarder is cluster scoped.
-  oc get clusterlogforwarder instance -o yaml > $outdir/clusterlogforwarder.instance.yaml  ||:
-
-  oc -n ${LOGGING_NS} extract secret/elasticsearch --to=$outdir ||:
-  oc -n ${LOGGING_NS} extract configmap/fluentd --to=$outdir ||:
-  oc -n ${LOGGING_NS} extract configmap/secure-forward --to=$outdir ||:
-  oc -n ${LOGGING_NS} extract secret/secure-forward --to=$outdir ||:
-
-  oc -n ${LOGGING_NS} describe ds/fluentd > $outdir/fluentd.describe.txt  ||:
-  
-
-  oc -n openshift-logging exec -c elasticsearch \
-    $(oc -n openshift-logging get pods -l component=elasticsearch -o jsonpath={.items[0].metadata.name}) \
-    -- indices > $outdir/indices.txt||:
-  for p in $(oc -n openshift-logging get pods -l component=fluentd -o jsonpath={.items[*].metadata.name}); do
-    oc -n openshift-logging exec -- ls -l /var/lib/fluentd/clo_default_output_es > $outdir/$p.buffers.txt||:
-    oc -n openshift-logging exec -- ls -l /var/lib/fluentd/retry_clo_default_output_es > $outdir/$p.buffers.retry.txt||:
-  done
-
-  oc -n openshift-operators-redhat get deployment elasticsearch-operator -o wide > $outdir/eo-deployment.txt 2>&1 || :
-  oc -n openshift-operators-redhat describe deployment elasticsearch-operator > $outdir/eo-deployment.describe 2>&1 || :
-  oc -n openshift-operators-redhat logs deployment/elasticsearch-operator > $outdir/elasticsearch-operator.logs 2>&1 || :
-
-  get_all_logging_pod_logs $outdir
-  get_all_olm_logs $outdir
+  oc adm must-gather --image=${IMAGE_CLUSTER_LOGGING_OPERATOR:-quay.io/openshift/origin-cluster-logging-operator:latest} --dest-dir=$outdir
```

Comment:
does `oc adm must-gather` generate a zip file which must be downloaded, or will put files in the `$outdir` folder?
if it will generate a zip file,  can we add a zip extract step which makes links so that files can be viewwd from browser as before?

Reply:
@vimalk78 `oc adm must-gather` does not generate any zip file for your. It places all the files in `$outdir`.